eslintrc.json

```diff
@@ -0,0 +1,17 @@
+{
```

Comment:
Why was this renamed? Doesn't `eslint` read `.eslintrc`?

Reply:
https://eslint.org/docs/user-guide/migrating-to-7.0.0#runtime-deprecation-warnings-for-eslintrc-config-files

Reply:
There are few more options but json is one of suggested

Reply:
That link explicitly says to add a `.eslintrc` file to the project. By default, `eslint` will look for `.eslintrc` in the project directory (the parent directory when running the tool). So let's keep it `.eslintrc` and remove the `--config` in the npm script.